app/assets/stylesheets/_variables.scss

```diff
@@ -132,3 +142,38 @@ $font_color_select2_container: $colors_dark;
 /* - misc */
 $bg_color_descriptions: $colors_lightbrown; // description subheadings on templates page ("single-description"), *not* posts
 $bg_color_selectionpopup: $colors_lightbrown; // selection popups like character/alias on posts#editor
+$bg_color_spacer_alt: $colors_brown;
+$bg_color_search_box: $colors_brown;
+
+/* - replies.scss */
+$bg_color_icon_overlay: #000000; // box displaying current icon
+
+$bg_color_expander: #B0BBA9; // expander on posts
+
+$bg_color_ender_post: #3E384F;
+$font_color_ender_post: $colors_bright;
+
+$bg_color_post_character: $colors_brown;
+$bg_color_post_screenname: $colors_greybrown;
+$bg_color_post_author: $colors_lightbrown;
+
+$bg_color_post_character_narrow: $colors_lightbrown;
+$bg_color_post_author_narrow: $colors_brown; // these are flipped from not-narrow mode to make the quick-switcher blend in
+
+$font_color_paginate_link: #EEEEEE;
+$bg_color_paginate_link_hover: $colors_greybrown;
+$font_color_paginate_link_visited: $colors_brown;
+$bg_color_page_current: $colors_green;
+$font_color_page_disabled: #101010;
+$bg_color_post_icon: #B0BBA9;
+
+$font_color_menu_link: #EEEEEE;
+$bg_color_menu_link_hover: #B0BBA9;
```

Comment:
the colors in the semantic part that haven't been pulled out into the palette bit yet should be, as in the constellation dev chat:

```scss
// new colors
$colors_blue: #4455AA;
$colors_red: #880000;
$colors_greygreen: #B0BBA9;
$colors_lightgrey: #EEEEEE;
$colors_darkpurple: #3E384F;
$colors_black: #101010;
```